src/redisearch_rs/inverted_index_bencher/build.rs

```diff
@@ -8,48 +8,23 @@
 */
 
 use std::env;
-use std::path::{Path, PathBuf};
+use std::path::PathBuf;
 
-use build_utils::{git_root, rerun_if_c_changes};
+use build_utils::{git_root, link_static_libraries, rerun_if_c_changes};
 
 fn main() -> Result<(), Box<dyn std::error::Error>> {
     // Always link the static libraries, independent of bindgen
-    link_static_libraries();
+    link_static_libraries(&[
+        ("src/inverted_index", "inverted_index"),
+        ("src/buffer", "buffer"),
+    ]);
 
     // Generate C bindings - fail build if this doesn't work
     generate_c_bindings()?;
 
     Ok(())
 }
 
-fn link_static_libraries() {
```

Comment:
Doesn't some of the other `build.rs` files also have this already?

Reply:
No, I grepped for `rustc-link-lib` and `rustc-link-search`, that's the only ones.